cmd/verify/verify.go

```diff
@@ -40,40 +55,110 @@ func main() {
 	forever := make(chan bool)
 
 	log.Info("starting verify service")
-	var message Message
-	// var header []byte
 
 	go func() {
 		for d := range broker.GetMessages(mq, config.Broker.Queue) {
 			log.Debugf("received a message: %s", d.Body)
-			err := json.Unmarshal(bytes.Replace(d.Body, []byte(`'`), []byte(`"`), -1), &message)
-			if err != nil {
+			var m Message
+			e := json.Unmarshal(bytes.Replace(d.Body, []byte(`'`), []byte(`"`), -1), &m)
+			if e != nil {
 				log.Errorf("Not a json message: %s", err)
+				// Nack errorus message so the server gets notified that something is wrong but don't requeue the message
+				d.Nack(false, false)
```

Comment:
Here and below you are ignoring error from Nack() method. shouldn't it be processed somehow (at least, logged)? If you don't want to do anything about it, ignore it explicitly with `_`, following the Go style.

Reply:
Strange that that one didn't raise a flag, normally the code tools will flag that sort of stuff.

Reply:
Which IDE do you use? I use Goland and it highlights it :)